src/lib/firebase.js

```diff
@@ -12,9 +11,9 @@ const firebaseConfig = {
   storageBucket: 'tcl-2-smart-shopping-list.appspot.com',
   messagingSenderId: '538706887514',
   appId: '1:538706887514:web:30b4235c1e8aa5fa19a963',
+  timestampsinSnapshots: true,
 };
 
 const fb = firebase.initializeApp(firebaseConfig);
 const db = fb.firestore();
-
```

Comment:
Is there a reason you're deleting lines here?

Reply:
We did not mean to delete any lines. Since Ethan and I got started working on our branch after you guys had already made some good changes, we decided to copy and paste all your new code into our branch. That was not a good decision and that is why our branch is all over the place. Sorry for the confusion! We will keep your code when we merge.